_source/_posts/2022-09-12-react-native-spring-boot-jhipster.adoc

```diff
@@ -0,0 +1,531 @@
+---
+layout: blog_post
+title: "Integrate React Native and Spring Boot Securely"
+author: matt-raible
+by: advocate
+communities: [mobile,java,javascript]
+description: ""
+tags: []
+tweets:
+- ""
+image:
+type: conversion
+github: https://github.com/oktadev/auth0-react-native-jhipster-example
+---
+:page-liquid:
+:toc: macro
+:experimental:
+:commandkey: &#8984;
+
+// If this post is not targeting keywords, "Get started with React Native for JHipster" is probably a better title.
+//   - 55: Integrate React Native and Spring Boot Securely
+//   - 57: React Native Authentication with OAuth and Spring Boot
+//   - 60: Secure React Native and Spring Boot with OpenID Connect
+//   - 61: Build React Native Apps Quickly with JHipster
+//   - 65: Build Secure React Native Apps Quickly with JHipster
```

Comment:
We could also match the [Ionic blueprint post](https://developer.okta.com/blog/2022/05/12/ionic-angular-jhipster) from May. That is, use `react-native-jhipster` for the URL and `Build Secure React Native Apps with JHipster` (62) for the title. The Ionic blueprint post hasn't done very well in terms of popularity. Since launch it only has 960 sessions. However, 577 of those are new users.